src/main/java/org/alliancegenome/curation_api/model/entities/GeneDiseaseAnnotation.java

```diff
@@ -36,7 +36,7 @@
 @AGRCurationSchemaVersion(min = "1.2.0", max = LinkMLSchemaConstants.LATEST_RELEASE, dependencies = { DiseaseAnnotation.class })
 public class GeneDiseaseAnnotation extends DiseaseAnnotation {
 
-	@IndexedEmbedded(includeDepth = 1)
+	@IndexedEmbedded(includePaths = { "geneSymbol.displayText", "geneFullName.displayText", "geneSystematicName.displayText", "geneSynonyms.displayText" })
```

Comment:
The only issue which I assume is obvious is that if more fields are needed then they have to be added here by doing it this way. Otherwise just set the includeDepth = 2 and capture everything.

Reply:
If you set depth=2 ElasticSearch throws an error about too many fields (at least locally)

Reply:
`"error": {
    "root_cause": [
      {
        "type": "illegal_argument_exception",
        "reason": "Limit of total fields [1000] has been exceeded"
      }
    ],
    "type": "illegal_argument_exception",
    "reason": "Limit of total fields [1000] has been exceeded"
  },`